chromium/browser_type.go

```diff
@@ -177,7 +177,7 @@ func (b *BrowserType) allocate(
 	opts *common.LaunchOptions,
 	flags map[string]interface{},
 	env []string,
-) (_ *common.BrowserProcess, rerr error) {
+) (_ *common.BrowserProcess, rErr error) {
```

Comment:
I'd keep this as it was, as it's a common convention in Go to name short error variables as `rerr` instead of `rErr`. For example, see https://github.com/golang/go/search?q=rerr.

Reply:
👍